index.js

```diff
@@ -184,7 +184,7 @@ async function httpProxy (fastify, opts) {
   })
 
   function handler (request, reply) {
-    let dest = request.raw.url
+    let [dest] = request.raw.url.split('?')
```

Comment:
Please avoid split and destructuring, they are slow. Identify the index of ?, and use slice.